### ISSUE TITLE
NansException: A tensor with all NaNs was produced in Unet 

### ISSUE THREAD

**Issue by barabasj** [bmaltais/kohya_ss#1059]
NansException: A tensor with all NaNs was produced in Unet after last update using just created Dreambooth LoRA in automatic1111.

Complete Message:
NansException: A tensor with all NaNs was produced in Unet. This could be either because there's not enough precision to represent the picture, or because your video card does not support half type. Try setting the "Upcast cross attention layer to float32" option in Settings > Stable Diffusion or using the --no-half commandline argument to fix this. Use --disable-nan-check commandline argument to disable this check.

Previous created Dreambooth LoRA's models (just two days ago) still work OK. The LoRA's process creation produced no errors but it finished in only 20 minutes aprox instead of the normal 60 minutes that always takes. Parameters used have not changed.

Parameters:

LORA type - Standard
Train Batch - 2
Epoch - 1
Caption Extension - .txt
Mixed Precision - fp16
Save Precision - fp16
Seed - 1234
Learning rate - 0.0001
LR Scheduler - Constant
LR Warmup - 0
Optimizer - AdamW8bit
Network Rank - 128
Network Alpha - 128
Enable Buckets - YES 

.... in Advanced Configuration ....
- 
CLIP SKIP - 2
Max Token Length - 75
Bucket Resolution Steps - 64

I'm running with a RTX 3060 12 Gig.
Regards.

**Comment by bmaltais** [comment]
Hard to troubleshoot without the full JSON config and source dataset

**Comment by barabasj** [comment]
I run the process twice with same results. Using the same input images and caption files. So, it can be reproduced, at least, in my installation (Windows), where it worked ok until now. Which files do you need? The process did not issued any errors during Dreambooth LoRA creation. I've noticed that the process was VERY fast 20 min instead of the 60 usual.
Attaching log file produced the last time.
[20230624144514.zip](https://github.com/bmaltais/kohya_ss/files/11857979/20230624144514.zip)

**Comment by bmaltais** [comment]
Ahhh... You are using the 8bit optimizer... I suspect the new bitsandbytes-windows is the cause. I will return to the original as the default since it is probably the cause...

**Comment by barabasj** [comment]
Great! Thanks.

**Comment by bmaltais** [comment]
The new version will install the previous bitsandbytes==0.35.0 and the dll patches. Should work back again like it used to. I suggest deleting the venv and doing a full setup to make sure things are in full order.

**Comment by barabasj** [comment]
After installing from scratch it is working again. Thanks!
